src/App.css

```diff
@@ -11,67 +24,60 @@ li {
 }
 
 .nav-wrap p {
-  color: white;
-  font-size: 30px;
-  font-weight: 600;
   margin-left: 15px;
+  font-size: 1.875rem;
+  font-weight: 800;
+  color: rgb(2, 144, 255);
+}
+
+.book-list {
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.book-list li {
+  width: 100%;
+  padding: 10px;
 }
 
 .nav-wrap ul {
   display: flex;
   justify-content: flex-end;
-  margin-right: 40px;
+}
+
+.Book-container li {
+  display: flex;
+  align-items: center;
 }
 
 .nav-wrap ul li {
   list-style: none;
   margin: 10px 30px;
-  color: white;
+  color: black;
 }
 
 .book-page {
   width: 100%;
-  background-color: yellowgreen;
-  height: 100vh;
+  height: 100%;
   display: flex;
   justify-content: center;
-}
-
-.book-form {
-  display: flex;
-  flex-direction: column;
+  flex-direction: column-reverse;
   align-items: center;
-  justify-content: center;
```

Comment:
- [ ] Small thing is missing on the form please implement i. The title is missing `ADD NEW BOOK` Please implement it. Also please use the same color for the placeholder in the input field. And Fix the padding for Add book. See image below for reference
### Your design
![his](https://user-images.githubusercontent.com/81727988/222903971-60847b6e-c9b5-43b2-a89b-8d29ecfd1598.png)

### Original design
![original](https://user-images.githubusercontent.com/81727988/222903980-13b0a181-0f8f-43a0-9c69-8e1ead9480db.png)